resource/mail/acc32_mail_secret2_german.txt

```diff
@@ -1,7 +1,7 @@
 GAME
 {
 	"MailFrom1"	"Von: DJH"
-	"MailDate1"	"Datum: 6. Juli 2222"
+	"MailDate1"	"Datum: 6. Juli 2022"
```

Comment:
good catch. this one is a fourth wall break (the mail was written in the real world by the mapper)